.changesets/diagnose-report-now-gets-the-agent-version-from-the-agent-file.md

```diff
@@ -0,0 +1,6 @@
+---
+bump: "patch"
+type: "change"
+---
+
+The diagnose report now gets the agent version from the agent file.
```

Comment:
```suggestion
The diagnose library report now reports the agent version from the committed agent file, rather than the downloaded version, which is reported in the installation report.
```